### PR TITLE
DYN-5876 update benchamark test suite

### PR DESCRIPTION
**Pull request by Cristhian-Marin-asdk**
### Purpose
DynamoPerformanceTests project was using obsolete project structure, now it's converted to SDK project structure

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated
- [x] This PR contains no files larger than 50 MB 

### Release Notes

Converted DynamoPerformanceTests.csproj to SDK project structure

### CODE REVIEW

**Comment by JimmySanchezGlobant** [comment]
LGTM

File: tools/Performance/DynamoPerformanceTests/DynamoPerformanceTests.csproj [hunk 1]
```diff
@@ -1,259 +1,50 @@
...
+            <Link>Properties\AssemblySharedInfo.cs</Link>
+        </Compile>
+    </ItemGroup>
+
+    <ItemGroup>
+        <PackageReference Include="BenchmarkDotNet" Version="0.11.4" />
+        <PackageReference Include="Microsoft.CodeAnalysis" Version="4.6.0" />
+        <PackageReference Include="Microsoft.CSharp" Version="4.7.0" />
+        <PackageReference Include="Microsoft.VisualBasic" Version="10.3.0" />
+        <PackageReference Include="NDesk.Options.Patched" Version="0.3.2" />
```
Comment (mjkkirschner) [inline tools/Performance/DynamoPerformanceTests/DynamoPerformanceTests.csproj:38]: could you align this with commandline parser? We've tried to remove ndesk options from the repo.

File: tools/Performance/DynamoPerformanceTests/DynamoPerformanceTests.sln [hunk 2]
```diff
@@ -1,20 +1,38 @@
...
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "DynamoPerformanceTests", "DynamoPerformanceTests.csproj", "{B19C56FE-48C0-445E-9200-2CBF9B6AC77F}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DynamoPerformanceTests", "DynamoPerformanceTests.csproj", "{F30253B6-6F68-4E56-9358-0832E4F38A0F}"
 EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
+		Debug|NET60_Linux = Debug|NET60_Linux
+		Debug|NET60_Windows = Debug|NET60_Windows
+		Debug|x64 = Debug|x64
```
Comment (mjkkirschner) [inline tools/Performance/DynamoPerformanceTests/DynamoPerformanceTests.sln:13]: I don't think there is a reason to add a new x64 platform

Comment (Cristhian-Marin-asdk) [reply tools/Performance/DynamoPerformanceTests/DynamoPerformanceTests.sln:13]: Ok, I will remove it

File: tools/Performance/DynamoPerformanceTests/DynamoPerformanceTests.csproj (hunk 1 above)
Comment (Cristhian-Marin-asdk) [reply tools/Performance/DynamoPerformanceTests/DynamoPerformanceTests.csproj:38]: Yes I'll work on that

**Comment by mjkkirschner** [comment]
The failing test is unrelated, LGTM.

**Review by mjkkirschner — APPROVED** [review]

**Review by mjkkirschner — APPROVED** [review]

**Pull request merged**